benchmarking/evaluations/scenarios/concentrate_timeslot_diversify_gender_min_2.py

```diff
@@ -0,0 +1,49 @@
+from typing import List
+
+from api.models.enums import (
+    DiversifyType,
+    ScenarioAttribute,
+    Gender,
+    TokenizationConstraintDirection,
+    Race,
+)
+from api.models.tokenization_constraint import TokenizationConstraint
+from benchmarking.evaluations.goals import WeightGoal, DiversityGoal
+from benchmarking.evaluations.interfaces import Scenario, Goal
+
+
+class ConcentrateTimeslotAndDiversifyGenderMin2Female(Scenario):
+    def __init__(
+        self,
+        max_num_choices: int,
+        value_of_female: int = Gender.FEMALE,
```

Comment:
this default will fail, Gender.FEMALE is not an int

This comment applies to all defaults like this